lib/metasploit/framework/login_scanner/redis.rb

```diff
@@ -20,6 +20,8 @@ class Redis
         LIKELY_SERVICE_NAMES = [ 'redis' ]
         PRIVATE_TYPES        = [ :password ]
         REALM_KEY            = nil
+        OLD_PASSWORD_UNSET   = /but no password is set/i
+        PASSWORD_UNSET       = /without any password configured/i
```

Comment:
Might just be me but when I read `unset` I think more that something has been set and then it was unset, I think `NOT_SET` might read better?
I'm also not really sure I understand the nuances between `PASSWORD_UNSET` and `OLD_PASSWORD_UNSET` what's the difference between "no password is set" and "without any password configured"